src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -0,0 +1,34 @@
+package racingcar.view
+
+import racingcar.domain.RacingCar
+
+class OutputView {
+    fun printGettingCarsName() {
+        println(INPUT_CARS_NAME)
+    }
+
+    fun printGettingRoundCount() {
+        println(INPUT_ROUND_COUNT)
+    }
+
+    fun printResult() {
+        println(OUTPUT_RESULT)
+    }
+
+    fun printEachRound(cars: List<RacingCar>) {
+        cars.forEach { println(it.toString()) }
+        println()
+    }
+
+    fun printWinner(winner: List<String>) {
+        println("$OUTPUT_WINNER${winner.joinToString(separator = ", ")}")
+    }
+
+    companion object {
+        const val INPUT_CARS_NAME = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)."
```

Comment:
```suggestion
        private const val INPUT_CARS_NAME = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분)."
```
클래스 내부에서만 사용하는 것이라면, private 접근자를 붙여보면 어떨까요? (이하 관련 내용 동일)